Resources/doc/index.rst

```diff
@@ -293,3 +293,93 @@ fixture using the ``UserFixtures`` group:
 .. _`installation chapter`: https://getcomposer.org/doc/00-intro.md
 .. _`Symfony Flex`: https://symfony.com/doc/current/setup/flex.html
 .. _`default service configuration`: https://symfony.com/doc/current/service_container.html#service-container-services-load-example
+
+
+Specifying purging behavior
+---------------------------
+
+By default alls previously existing data is purged using ``DELETE FROM table`` statements. If you prefer to use
+``TRUNCATE table`` statements for purging, use ``--purge-with-truncate``.
+
+If you want to exclude a set of tables from being purged, e.g. because your schema comes with pre-populated,
+semi-static data, pass the option ``--purge-exclusions``. Specify ``--purge-exclusions`` multiple times to exclude
+multiple tables.
+
+You can also customize purging behavior significantly more and implement a custom purger plus a custom purger factory.
+
+
+    // src/Purger/CustomPurger.php
+    namespace App\Purger;
+
+    use Doctrine\Common\DataFixtures\Purger\PurgerInterface;
+
+    // ...
+    class CustomPurger implements PurgerInterface
+    {
+        public function purge(): void
+        {
+            // ...
+        }
+    }
+
+    // src/Purger/CustomPurgerFactory.php
+    namespace App\Purger;
+    // ...
+    use Doctrine\Bundle\FixturesBundle\Purger\PurgerFactory;
+
+    class CustomPurgerFactory implements PurgerFactory
+    {
+        public function createForEntityManager(?string $emName, EntityManagerInterface $em, array $excluded = []) : PurgerInterface;
```

Comment:
```suggestion
        public function createForEntityManager(?string $emName, EntityManagerInterface $em, array $excluded = []): PurgerInterface;
```

1. to be consistent with the style you used above
2. we are going to remove that space in the next version of our coding standard.